packages/util-package-renderer/lib/js/js-helper.js

```diff
@@ -30,9 +35,21 @@ const transpileJS = async (packageRoot, demoCodeFolder) => {
 	}
 
 	// Create a rollup bundle
+	// - Handle import and require
+	// - Transpile using babel
 	try {
 		bundle = await rollup.rollup({
 			input: path.join(packageRoot, demoCodeFolder, 'main.js'),
+			plugins: [commonjs({
```

Comment:
```suggestion
			plugins: [
						commonjs({
```
etc.
because it took me ages to figure out the `nodeResolve(),` line 😂